src/main/steps/sequence/have-you-applied-for-help-with-fees/content.ts

```diff
@@ -26,8 +26,27 @@ export const generateContent: TranslationFn = ({ isDivorce }) => {
     },
   };
 
-  // @TODO translations
-  const cy: typeof en = { ...en };
+  const cy: typeof en = {
+    title: `Ydych chi eisoes wedi gwneud cais am help i dalu ${isDivorce ? 'ffi eich ysgariad' : 'eich ffi'}?`,
+    yes: 'Do',
```

Comment:
i think we should move this to the comment content welsh version and remove them here include the english one